website/docs/add-secure-apps/providers/ssf/index.md

```diff
@@ -1,30 +1,45 @@
 ---
 title: Shared Signals Framework (SSF) Provider
 sidebar_label: SSF Provider
+tags: [ssf, backchannel, provider, "security event tokens"]
 authentik_version: "2025.2.0"
 authentik_enterprise: true
 authentik_preview: true
 ---
 
-Shared Signals Framework (SSF) is a common standard for sharing asynchronous real-time security signals and events across multiple applications and an identity provider. The framework is a collection of standards and communication processes, documented in a [specification](https://openid.net/specs/openid-sharedsignals-framework-1_0-ID3.html). SSF leverages the APIs of the application and the IdP, using privacy-protected, secure webhooks.
+Shared Signals Framework (<abbr>SSF</abbr>) is a collection of standards for sharing asynchronous security signals and events across multiple applications and an identity provider. SSF leverages the APIs of the application and the IdP, using privacy-protected, secure webhooks.
```

Comment:
@Teffen, as I indicated in an earlier review (apparently dismissed) I want to **leave the link to the spec where it is mentioned** in this doc. Please do not make any of the changes in this section. Actually, please remove this file from this PR. The PR was intended to create the ABM integration docs. If you'd like to change the SSF Provider docs (and the several other files now included in this PR), please open a separate PR. Let's practice good PR hygiene.